docs/modules/ROOT/pages/tests-configuration.adoc

```diff
@@ -250,11 +250,6 @@ bdd.all-meta-filters=groovy: !this['test-stand']
 |`desktop`, `phone`, `phone_portrait`, `phone_landscape`, `chrome_phone`, `chrome_responsive`, `chrome_tablet`, `tablet`, `tablet_landscape`, `tablet_portrait`
 |Used to specify layout, using which marked story or scenario should be executed. In case if this meta tag is not specified, default (`desktop`) value is used for web applications.
 |`@layout desktop`
-
-|@browserWindowSize
```

Comment:
Unfortunately we can't remove the feature without deprecation. The flow is
- deprecate old feature (log notifications for users, add annotations and et.c) and introduce new feature
- release, notify users in release notes
- old feature should survive several releases
- remove old feature in new major version or in new minor version (if major version number is `0`)

Reply:
Please, provide an example how this could be done - `log notifications for users, add annotations and et.c`
I mean besides WARNING in documentation.

Reply:
https://github.com/vividus-framework/vividus/commit/83659464540ccd87cb891287eee673a70c26cec6